lib/src/emoji_cell.dart

```diff
@@ -99,16 +101,39 @@ class EmojiCell extends StatelessWidget {
     );
   }
 
-  /// Build and display Emoji centered of its parent
-  Widget _buildEmoji() {
-    final style = TextStyle(
+  static String? _getPlatformDefaultEmojiFont() {
+    if (Platform.isIOS || Platform.isMacOS) {
+      return 'Apple Color Emoji';
```

Comment:
can be a const

Reply:
Sorry, I'm not sure what you mean. I don't think `return const 'Apple Color Emoji';` is valid syntax.

Maybe you mean that I should define a constant somewhere for this string literal?

Reply:
I mean you can define it as a constant outside of the class and then use it.
Like some where at the top
`const AppleEmojiFont = 'Apple Color Emoji';`
and then here at return you use it `return AppleEmojiFont`